examples/key_events.rs

```diff
@@ -94,17 +94,18 @@ pub fn read_asynchronously() {
         }
 
         // disable mouse events to be captured.
-        input.disable_mouse_mode().unwrap();
+        input.disable_mouse_mode()?;
     } // <=== raw modes will be disabled here
+    Ok(())
 } // <=== background reader will be disposed when dropped.
 
-pub fn read_synchronously() {
+pub fn read_synchronously() -> Result<()> {
     // make sure to enable raw mode, this will make sure key events won't be handled by the terminal it's self and allows crossterm to read the input and pass it back to you.
-    if let Ok(_raw) = RawScreen::into_raw_mode() {
+    if let Ok(_) = RawScreen::into_raw_mode() {
```

Comment:
raw screen will be disabled when dropped, so we probably don't want to do this. Or isn't it dropped if you ignore the variable

Reply:
Good catch, fixed.